### DISCUSSION TITLE
When Opening a Source File, I get the attached error

### DISCUSSION THREAD

**Discussion by edshaw42** [codefori/vscode-ibmi#487] [General]
Hello,

I didn't want to create an issue yet incase it was just me. It's been about 3 months since I have used this, but when I create a filter for my library/sourcepf and then expand the source file I get the following error every time:

![Screen Shot 2022-01-28 at 8 59 19 AM](https://user-images.githubusercontent.com/85707702/151569515-3862bd4a-75fc-43aa-be6c-3fb5b7646f6f.png)

Help?

**Comment by worksofliam** [comment]
@edshaw42 Do you have `db2util` installed on your IBM i? It could likely be that that is causing the issue. 

If you do, you could temporarily uninstall it and give that a try.

**Reply by worksofliam** [reply]
Also, as of 0.8.26 (which just came out), instead of uninstalling db2util you can just change the SQL executer from `default` to something else.

I'd recommend you change it to `db2` or `none` and then try and load the member list again. Perhaps we have found an issue with db2util.

![image](https://user-images.githubusercontent.com/3708366/151576504-d0889c4a-593f-459a-8177-179516ca19f0.png)

**Reply by worksofliam** [reply]
Lastly, If you did have db2util, I would make sure you have the latest version of db2util, which is 1.0.12

**Reply by edshaw42** [reply]
@worksofliam  Thank you!
